www/page_data.inc

```diff
@@ -69,9 +69,16 @@ function loadPageRunData($testPath, $run, $cached, &$requests = null, $options =
                 "loadEventEnd"
             );
             for ($i = 0; $i < count($ret); $i++) {
+
+              // are relative to the performance timing (navitation start)

```

Comment:
Can you elaborate? The Chrome extension will use the performance timing API to set the start time, more precisely the navigationStart timestamp.